README.md

```diff
@@ -17,6 +17,14 @@ _Leveraging reproducible research_
 bower install
 ```
 
+## Run
+
+```bash
+docker build --tag platform .
+
+docker run -d -p 80:80 platform
+```
+
 ## Configure
 
 Create a copy of the file `client/app/config/configSample.js` and name it `client/app/config/config.js`. You must configure the required application settings in this file, which is not part of the version control:
```

Comment:
So far, it also worked if I deleted the config as there are also predefined settings.